udata/core/dataset/search.py

```diff
@@ -45,8 +45,11 @@ class DatasetSearch(ModelSearchAdapter):
 
     @classmethod
     def is_indexable(cls, dataset):
```

Comment:
We don't need to modify this anymore, see https://github.com/opendatateam/udata/pull/2997